cypress/integration/DiseaseOnly/DiseaseCodeUglyToPrettyURL.feature

```diff
@@ -5,3 +5,9 @@ Feature: As the system, I want to be able to redirect users to pretty URLs for t
 		And "pageTitle" is set to "{{disease_label}} Clinical Trials"
 		Given the user navigates to "/C4872"
 		Then the user is redirected to "/breast-cancer?redirect=true"
+
+	Scenario: Page redirects to pretty URL if pretty URL name parameter is given, without appending redirect=true
```

Comment:
Can you label this accordingly. Scenario described doesn't match the steps below.

Reply:
Will do